client/my-sites/email/email-providers-stacked-comparison/email-provider-stacked-card/style.scss

```
@@ -204,16 +142,17 @@ h2.action-panel__title {
 		margin-top: 1em;
 		line-height: 1.5;
 
-		@include break-mobile {
+		//This needs to use deprecated breakpoint as we are using "UseBreakpoing" from "@automattic/viewport"
+		@include breakpoint-deprecated( '>660px' ) {
 			flex-direction: row;
 		}
 	}
 
 	.email-provider-stacked-card__provider-form {
 		margin-bottom: 20px;
-		width: 100%;
 
-		@media screen and ( min-width: 481px ) {
+		//This needs to use deprecated breakpoint as we are using "UseBreakpoing" from "@automattic/viewport"
```

Nit: Please make this more accurate. We're using `breakpoint-deprecated` in this code, and we're using the `useBreakpoint` hook from the `@automattic/viewport` package.
```suggestion
		// We need to use breakpoint-deprecated() as we are using the useBreakpoint() hook from the @automattic/viewport package
```